IFC_Engine/Convert/Physical/FromIFC/Floor.cs

```diff
@@ -0,0 +1,47 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2021, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.oM.Adapters.IFC;
+using BH.oM.Base;
+using BH.oM.Physical.Elements;
+using System.Collections.Generic;
+using Xbim.Ifc2x3.Interfaces;
+
+namespace BH.Engine.Adapters.IFC
+{
+    public static partial class Convert
+    {
+        /***************************************************/
+        /****              Public Methods               ****/
+        /***************************************************/
+
+        public static Floor FloorFromIFC(this IIfcSlab element, IFCSettings settings)
+        {
+            //TODO: refine this!
```

Comment:
Has an issue been raised on this?
Also what do you want to refine? Nothing is happening! 🍷

Reply:
Issue to be raised once this gets merged, see [this comment](https://github.com/BHoM/IFC_Toolkit/pull/4#issuecomment-844030444)